contracts/protocol/libraries/helpers/Errors.sol

```diff
@@ -66,6 +66,7 @@ library Errors {
   string public constant LPC_INVALID_ADDRESSES_PROVIDER_ID = '40'; // 'The liquidity of the reserve needs to be 0'
   string public constant LPC_INVALID_CONFIGURATION = '75'; // 'Invalid risk parameters for the reserve'
   string public constant LPC_CALLER_NOT_EMERGENCY_ADMIN = '76'; // 'The caller must be the emergency admin'
+  string public constant LPC_CALLER_NOT_POOL_OR_PROOF_OF_RESERVE_ADMIN = '81'; // 'The caller must be the pool or proof of reserve admin'
```

Comment:
why are the errors not sorted by number here (and with the two above?)

Reply:
they seem to be sorted by meaning/group